### PR TITLE
Redirect Scarlets and Blues to fe-project

### PR DESCRIPTION
**Pull request by eatyourgreens**
Redirect `bogden/scarlets-and-blues` to fe-project.zooniverse.org.

Staging branch URL: https://pr-6001.pfe-preview.zooniverse.org

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Review by shaunanoordin — APPROVED** [review]
## PR Review

This PR is part of what's required to make the [Scarlets and Blues project](https://www.zooniverse.org/projects/bogden/scarlets-and-blues) use FEM instead of PFE.

- ⚠️ External dependency: Doorkeeper system needs to be updated accordingly

### Status

LGTM, this is a pretty standard change. Might be best to merge only when Doorkeeper has been updated, though.

**Comment by eatyourgreens** [comment]
The proxy rules are in https://github.com/zooniverse/static/pull/242. I’ll merge these both together.

**Comment by eatyourgreens** [comment]
No, wait, there’s a typo in the URLs here.

File: app/router.cjsx [hunk 1]
```diff
@@ -178,7 +178,10 @@ module.exports =
 
     <Route path="/projects/hughdickinson/superwasp-black-hole-hunters" component={() => <RELOAD newUrl='https://fe-project.zooniverse.org/projects/hughdickinson/superwasp-black-hole-hunters' />} />
     <Route path="/projects/hughdickinson/superwasp-black-hole-hunters/classify" component={() => <RELOAD newUrl='https://fe-project.zooniverse.org/hughdickinson/superwasp-black-hole-hunters/classify' />} />
-    
+
+    <Route path="/projects/bogden/scarlets-and-blues" component={() => <RELOAD newUrl='https://fe-project.zooniverse.org/projects/bogden/scarlets-and-blues' />} />
+    <Route path="/projects/bogden/scarlets-and-blues/classify" component={() => <RELOAD newUrl='https://fe-project.zooniverse.org/bogden/scarlets-and-blues/classify' />} />
```
Comment (eatyourgreens) [inline app/router.cjsx:183]:
```suggestion
    <Route path="/projects/bogden/scarlets-and-blues/classify" component={() => <RELOAD newUrl='https://fe-project.zooniverse.org/projects/bogden/scarlets-and-blues/classify' />} />
```

**Comment by shaunanoordin** [comment]
Derp, I can't believe missed the missing `/projects/` as well. I need to be more careful to actually copy-paste those redirects to try them out next time.

**Pull request merged**